src/sharedTypes.ts

```diff
@@ -17,6 +17,8 @@ export interface Word {
   word: string;
   startTime: number;
   duration: number;
+  deleted: boolean;
```

Comment:
Why do we need to soft-delete words? Why can't we just remove them entirely?

Reply:
Let's say I deleted a section then made 4 edits after that. What if I want to restore the section I deleted 5 edits ago without having to revert all the way back to the state at that time? This way we at least have the option to restore deleted sections without needing to undo every change that came before it. 

It hasn't been implemented because the UI for the restore action is tricky - Zac doesn't want deleted text to have a strikethrough (which would be the easy way) so it needs some brainstorming before its implemented.

Reply:
Makes sense, I can see the value in this.

Are these requirements documented somewhere? I had never heard about it

Reply:
We had strikethrough in the original designs, Zac asked us to scrap it. I didn't want to fully remove the possibility - I'm going to try and convince him otherwise.

Reply:
Cool, is there an overall scope doc for the edit/delete stuff? Would be good to know what is and isn't being included

Reply:
The current scope is DWDD (do what Descript does). Happy to write something more formal up (and hopefully make it better then Descript)